src/vsphere_cpi/lib/cloud/vsphere/vm_config.rb

```diff
@@ -1,7 +1,15 @@
+require 'URI'
+
 module VSphereCloud
   class VmConfig
     include Logger
 
+    MAX_VM_NAME = 79
+    MAX_VM_NAME_IN_BYTE = 213
+    MAX_DEPLOYMENT_NAME = 25
```

Comment:
These three comments are basically implementation details of a private method and in my opinion don't need to be here.